inst/include/conditionalSampler.h

```diff
@@ -0,0 +1,673 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
```

Comment:
This could be removed.  It's how Adam and I (and many others....) used to tell Emacs about preferences on per-file basis.

These days we have the file `.editorconf` -- which does this project-wide, and for other editors.

Reply:
Agreed! I think I'll do it in #61 and then probably for all files (it's something I noticed in all files) ?

@adamjohansen do you need these type of lines for something?

Reply:
No, it's an artifact of the past as @eddelbuettel suggested. Sorry for being slow, I will look at things properly later today.